public/resources/package.json

```diff
@@ -7,20 +7,20 @@
     "start": "twilio-run --env"
   },
   "devDependencies": {
-    "twilio-run": "^2.0.0"
+    "twilio-run": "^2.0.0",
```

Comment:
I changed this based on Dominik's comment: "A lot of the dependencies should be devDependencies since there is no reason for them to be installed in the serverless environment"